### DISCUSSION TITLE
Unhide ElevenClock?

### DISCUSSION THREAD

**Discussion by PescheHelfer** [martinet101/ElevenClock#979] [Q&A] [answered]
When automatically hiding in full screen does not work, I use the "Hide ElevenClock" context menu option. However, I haven't figured out how to unhide ElevenClock again. How do you do that?

Thanks for any hints.

**Comment by marticliment** [comment, marked as answer]
You can:
1. single-click the taskbar icon tray
2. Right-click another clock -> reload clocks
3. Launch again ElevenClock from the start menu (slower)

**Comment by PescheHelfer** [comment]
Thanks, but I meant, when you right-click the clock and choose "Hide ElvenClock".
Option 1 and 2 won't work, because there is no ElvenClock anymore.
You can launch it again, but don't you have two instances then, one hidden and one visible?

**Comment by marticliment** [comment]
You are clicking hide ElevenClock or close ElevenClock?

Because a system tray ick shod be present if you haven't disabled it

**Comment by PescheHelfer** [comment]
Ah, sorry, my bad. I did use "hide", but I haven't realized it's in the hidden system tray icons, where I can use "restart". After restarting the clock, however, the position is off. It is too far to the right, so that the system clock is partially visible on the the left. That's not the case when ElevenClock starts regularly after a reboot.
Edit: That position problem seems to be unrelated to the restart. I also got it now with a fresh windows boot. Probably, the position was just shifted. I have adjusted it, and now everything is fine, even after a restart. Thanks!
